src/index.html

```diff
@@ -718,6 +763,14 @@ <h2 class="controls-title">Duration</h2>
             clearAnnotations();
           });
 
+          uv.on("mediaelementExtension.mediaMuted", function() {
+            console.log("muted");

```

Comment:
Would it make sense to set the checkbox state in the form to match the state of the running application in these event handlers to keep things in sync?